src/install/installMultiple.ts

```diff
@@ -409,7 +410,7 @@ async function install (
         cpu: pkg.cpu,
         os: pkg.os,
       },
-      sideEffectsCache: pkgResponse.body.sideEffectsCache,
+      cacheByNodeVersion: ctx.force ? '' : pkgResponse.body.sideEffectsCache[nodeMajor],
```

Comment:
you can make this property nullable and assign it null or undefined. The empty string doesn't make sense